workflow-templates/im-test-sam-yaml.properties.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "Test and Import SAM YAML",
```

If you're going with the test category the name should have test as the prefix so it shows up with all of the other test items when people go to add the workflow.  I wonder if something like Validate and Export sam.yaml metadata is a more specific description of what is happening.
```suggestion
  "name": "Test - Validate and Import SAM YAML",
```

---

> If you're going with the test category the name should have test as the prefix so it shows up with all of the other test items when people go to add the workflow. I wonder if something like Validate and Export sam.yaml metadata is a more specific description of what is happening.

I agree that "export" is a better description from the point of view of the repository where this workflow will run.

---

"Test - Validate and Export SAM YAML" sounds good to me.